requirements/base.txt

```diff
@@ -8,7 +8,7 @@ amqp==5.0.9
     # via kombu
 aniso8601==9.0.1
     # via tincan
-asgiref==3.4.1
+asgiref==3.5.0
```

Comment:
Dropped support for `Python 3.6`.  No further major change: https://github.com/django/asgiref/blob/main/CHANGELOG.txt